examples/cart/styles/_overlay.scss

```diff
@@ -0,0 +1,12 @@
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: none;
+
+  &.js-active {
+    display: block;
```

Comment:
In that case, we probably shouldn't be using an overlay at all - we should be listening globally for clicks, then ensuring they aren't within the cart element. Otherwise, people would be clicking on elements on the page to no effect. As an example,
```javascript
$(document).on('click', function(evt) {
  if(!$(evt.target).closest('.cart').length) {
    closeCart();
  }
});
```